app/models/volunteer.rb

```diff
@@ -165,8 +173,8 @@ def internal?
     !external
   end
 
-  def internal_and_assignments?
-    internal? && assignments?
+  def internal_and_started_assignments?
+    internal? && assignment_started? || group_assignment_stared?
```

Comment:
`internal? && (assignment_started? || group_assignment_stared?)`
`&&` [higher priority](has http://www.techotopia.com/index.php/Ruby_Operator_Precedence) than `||` so it executes first.